src/content/blog/ddev-name-resolution-wildcards.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "DDEV Name Resolution, Wildcards, Working Offline"
+pubDate: 2024-03-27
+#modifiedDate: 2024-03-03
+summary: How hostname name resolution works in a DDEV environment, including how wildcards work
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/03/2024-ddev.png
+  alt: Hostname resolution with DDEV
+  credit: 'Ideogram.ai: the words "2024" and "DDEV" next to each other'
+categories:
+  - Guides
+---
+
+A very cool thing about DDEV is that you don’t need to think about IP addresses or HTTPS or name resolution most of the time. Some of you will never need to. And you don't have to understand teh rest of this article to use DDEV.
```

Comment:
```suggestion
A very cool thing about DDEV is that most of the time you don’t have to worry about IP addresses, HTTPS or name resolution in general, some of you will never have to. The best thing, you don't even have to understand the rest of this article to use DDEV.
```
a list with two or in a row sounded odd. it sounded also odd "to think about ...". ive changed it to "worry about" and the second part was also hard to read for an introduction. and i picked up the pattern with the first sentence starting off with a very cool thing and going with the best thing for the second part.

Reply:
I like those changes. Might be my background, but I would change it to "IP addresses, SSL or DNS".